package.json

```diff
@@ -62,8 +62,8 @@
   "peerDependencies": {
     "classnames": ">=2",
     "prop-types": ">=15",
-    "react": ">=15",
-    "react-dom": ">=15",
+    "react": ">=16",
+    "react-dom": ">=16",
     "react-transition-group": "^2.5.0",
```

Comment:
let's update `react-transition-group` to `3.0.0` then as well

Reply:
Good catch. Updated